pkg/util/cert/io.go

```diff
@@ -117,3 +117,20 @@ func CertsFromFile(file string) ([]*x509.Certificate, error) {
 	}
 	return certs, nil
 }
+
+// PrivateKeyFromFile returns the private key in rsa.PrivateKey or ecdsa.PrivateKey format from a given PEM-encoded file.
+// Returns an error if the file could not be read or if the private key could not be parsed.
+func PrivateKeyFromFile(file string) (interface{}, error) {
+	if len(file) == 0 {
```

Comment:
fixed, I didn't understand why this was present in CertsFromFile from where I took it, removed now